### ISSUE TITLE
EditorConfig is missing

### ISSUE THREAD

**Issue by ghuntley** [Microsoft/TailwindTraders-Mobile#4]
Hey folks,

If this is going to be a reference app how about shipping Roslyn config via EditorConfig so more people discover the feature accidentally. Inconsistent formatting in a codebase within a team is maddening it doesn't have to be this way tho.

https://kent-boogaart.com/blog/editorconfig-reference-for-c-developers

**Comment by JimSEOW** [comment]
Was excited to try building this app to test the upcoming shell feature advertised through Visual Studio 2019. 
 Although I managed to compile using the recently released Visual studio 2019, it is not deployable to latest android emulator 8.1 API 27. After a closer look, there are just too many inconsistencies as stated above.  I hope it was a quick last minute effort by the team and that the app will soon be tested more extensively so we do not have to waste so much time finding out the inconsistencies that make the learning of the new Shell features challenging.

**Comment by MarcosCobena** [comment]
Hi @ghuntley,

I personally didn't know such before. I've had a look to the article and looks a good option.

We indeed tried to enforce coding guidelines through StyleCop Analyzers, and some others for the missing stuff: 120 chars/column, for instance. Also, as you noticed through Twitter, we tried to consider XAML as a first class citizen and at least provide cosmetic rules.

Thanks for pointing this out,

-Marcos
